### PR TITLE
Refactor logging gRPC server and its CLI

### PR DESCRIPTION
**Pull request by adibrastegarnia**
This PR  makes the required changes in logging gRPC service and its CLI.

### CODE REVIEW

**Comment by adibrastegarnia** [comment]
@kuujo 
Ready for your review whenever you get a chance.

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.6.14-dev
+0.6.14
```
Comment (kuujo) [inline VERSION:1]: I think we've decided not to do a release in the same PR as other changes... @ray-milkey

File: pkg/logging/logger.go [hunk 2]
```diff
@@ -42,6 +42,21 @@ func configure(config Config) error {
...
 
+// FindLogger finds a logger based on a given name
+func FindLogger(names ...string) Logger {
+	logger := root
+	for _, name := range names {
+		child, ok := logger.children[name]
+		if ok {
+			logger = child
+		} else {
+			return nil
```
Comment (kuujo) [inline pkg/logging/logger.go:53]: I don't necessarily think it's best to return `nil` if we can't find the logger. All loggers have a level, and if the client wants to ask about a theoretical logger that doesn't exist I think it's fine to do so and we can return the theoretical level of that theoretical logger. We should just be using `GetLogger` and returning the level of the logger the client asks for.

Comment (adibrastegarnia) [reply pkg/logging/logger.go:53]: Yeah. I was using Getlogger originally but I wasn't sure making this assumption about a theoretical logger is a valid assumption. I will use the GetLogger to do that but if we find it confusing then we can change it later

File: VERSION (hunk 1 above)
Comment (adibrastegarnia) [reply VERSION]: Done

File: pkg/logging/logger.go (hunk 2 above)
Comment (adibrastegarnia) [reply pkg/logging/logger.go]: Done

File: pkg/logging/server.go [hunk 3]
```diff
@@ -45,10 +45,39 @@ func (s Service) Register(r *grpc.Server) {
...
-		ResponseStatus: logging.ResponseStatus_OK,
+// GetLevel implements GetLevel rpc function to get a logger level
+func (s *Server) GetLevel(ctx context.Context, req *logging.GetLevelRequest) (*logging.GetLevelResponse, error) {
+
+	name := req.GetLoggerName()
+	if name == "" {
+		return &logging.GetLevelResponse{}, errors.New("precondition for get level request is failed")
+	}
+
+	names := strings.Split(name, "/")
```
Comment (kuujo) [inline pkg/logging/server.go:56]: I think we have a constant defined for `"/"`... `nameSep`

Comment (kuujo) [reply pkg/logging/server.go:56]: We could also move this `Split` call to a function.

Comment (adibrastegarnia) [reply pkg/logging/server.go:56]: OK. Let me fix it.

Comment (adibrastegarnia) [reply pkg/logging/server.go]: Done

**Pull request merged**